modules/apps/headless/headless-site/headless-site-impl/rest-openapi.yaml

```diff
@@ -34,9 +34,8 @@ components:
                     type: string
                 name:
                     type: string
-                parentSiteKey:
+                parentSiteERC:
```

Comment:
I think we should expand `ERC` and name it `parentSiteExternalReferenceCode`. This matches other usages within `rest-openapi.yaml` files.